docs/source/peer_ledger_snapshot.md

```diff
@@ -0,0 +1,153 @@
+Blockchain info: {"height":970,"currentBlockHash":"JgK9lcaPUNmFb5Mp1qe1SVMsx3o/22Ct4+n5tejcXCw=","previousBlockHash":"f8lZXoAn3gF86zrFq7L1DzW2aKuabH9Ow6SIE5Y04a4="}
+2018-02-25 15:15:44.139 UTC [main] main -> INFO 006 Exiting.....
+```
+
+In this example, the ledger height is `970`.
+
+A snapshot request can be submitted by issuing a command similar to:
+
+```
+peer snapshot submitrequest -C <name of channel> -b <ledger height where snapshot will be taken> --peerAddress <address of peer> --tlsRootCertFile <path to root certificate of the TLS CA>
+```
+
+For example:
+
+```
+peer snapshot submitrequest -C testchannel -b 1000 --peerAddress 127.0.0.1:22509 --tlsRootCertFile tls/cert.pem
+```
+
+**If you give a ledger height of `0`, the snapshot is taken immediately. This is useful for cases when an organization is interested in backing up its own ledger data and does not intend to share the data with another organization. Do not take these "immediate" snapshots in cases when snapshots will be evaluated from multiple peers, as it increases the likelihood that the snapshots will be taken at different ledger heights.**
+
+If the request is successful, you will see a `Snapshot request submitted successfully` message.
+
+You can list the pending snapshots by issuing a command similar to:
+
+```
+peer snapshot listpending -C testchannel --peerAddress 127.0.0.1:22509 --tlsRootCertFile tls/cert.pem
+```
+
+You will see a response similar to:
+
+```
+Successfully got pending snapshot requests [970]
+```
+
+When a snapshot has been generated for a particular block height, the pending request for that block height will no longer appear in the list. You can also verify that a snapshot has been create successfully by looking at the peer logs. You will see a message similar to `Generated snapshot block-<block number>`.
+
+To delete a snapshot request, simply exchange `submitrequest` with `cancelrequest`. For example:
+
+```
+peer snapshot cancelrequest -C testchannel -b 1000 --peerAddress 127.0.0.1:22509 --tlsRootCertFile tls/cert.pem
+```
+
+If you submit the `listpending` command again, the snapshot should no longer appear.
+
+### Contents of a snapshot
+
+Snapshots are stored on a peer in a snapshot folder that is further organized by channel name and the ledger height when the snapshot was taken: `{snapshotsFolder}/{channelName}/{blockHeight}`.
+
+As mentioned above, the snapshot contains four folders:
+
+* **Public state**
+  * This includes the latest state of all of the assets on the channel (for example, the current owner of a car).
+* **Private data hashes**
+  * Recall from our documentation on private data that while the actual transaction data of a private data transaction is not stored on the public ledger of the channel, hashes of the data are stored. This allows the private data to be verified against the hash, for example by an organization that is added to a private data collection.
```

Comment:
Better to add one line in the beginning - "This includes the private data hashes part of the world state of the channel."

Reply:
Added a note about this in the limitations section.

Reply:
The change in the limitations section LGTM. However, what I was not clear was whether you intend to add one liner here that explains what is included in the data under the heading (`Private data hashes`) or wanted to leave it with the heading being self explanatory. I am fine with either. But you want the latter, then the one liner can be removed from the previous heading  `Public state` as well for the consistency.